app/models/author_old.rb

```diff
@@ -0,0 +1,86 @@
+class Author
```

Comment:
Either remove this model or change the class name to AuthorOld. Don't want to risk the methods here overriding the ones in your new proper Author model.